pkg/morph/client/nns.go

```diff
@@ -48,7 +43,7 @@ var (
 // NNSAlphabetContractName returns contract name of the alphabet contract in NNS
 // based on alphabet index.
 func NNSAlphabetContractName(index int) string {
-	return "alphabet" + strconv.Itoa(index) + ".neofs"
```

Comment:
why `.neofs` has left us?

Reply:
Because https://pkg.go.dev/github.com/nspcc-dev/neofs-contract/rpc/nns#ContractReader.ResolveFSContract includes it.

Reply:
so this is not `contract name of the alphabet contract in NNS based on alphabet index.` right now?

Reply:
It's still a contract name. Not an FQDN, but a name.